installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="understanding-disconnected-installation-mirroring"]
+= Understanding disconnected installation mirroring
+include::_attributes/common-attributes.adoc[]
+:context: understanding-disconnected-installation-mirroring
+
+toc::[]
+// Reusing applicable content from Disconnected installation mirroring assembly
+
+You can use a mirror registry for disconnected installations and to ensure that your clusters only use container images that satisfy your organizational controls on external content. Before you install a cluster on infrastructure that you provision in a disconnected environment, you must mirror the required container images into that environment. To mirror container images, you must have a registry for mirroring.
```

Comment:
I'm also not sure what `organizational controls` refers to in this sentence. Do you mean organization like a company? If thats the case you can use `your organizations controls`, in place. That also just might be me being confused. 

```suggestion
You can use a mirror registry for disconnected installations to ensure that your clusters only use container images that satisfy your organizational controls on external content. Before you install a cluster on infrastructure that you provision in a disconnected environment, you must mirror the required container images into that environment. To mirror container images, you must have a registry for mirroring.
```

Reply:
your organization's controls>> sounds good , Implemented .